Command/CreateClientCommand.php

```diff
@@ -67,13 +69,28 @@ protected function configure(): void
                 InputArgument::OPTIONAL,
                 'The client secret'
             )
+            ->addOption(
+                'public',
+                null,
+                InputOption::VALUE_OPTIONAL,
```

Comment:
@X-Coder264 Is this suppose to have a value? If it's not a bool flag then how is it going to be used, eg
```
--public=0
--public=1
```
If that's the case I wouldn't add `false` as a default since it's not consistent. Maybe `'0'`?

Reply:
@HypeMC It's a bool flag.